keras_nlp/backend/config.py

```diff
@@ -65,7 +65,18 @@
 
 def detect_if_tensorflow_uses_keras_3():
     # We follow the version of keras that tensorflow is configured to use.
-    from tensorflow import keras
+    import keras
+
+    # Return False if env variable is set and `tf_keras` is installed.
+    use_legacy_keras = os.environ.get("TF_USE_LEGACY_KERAS", "")
+    if use_legacy_keras == "1" or use_legacy_keras.lower() == "true":
```

Comment:
We should probably match this https://github.com/tensorflow/tensorflow/blob/907196b966ba53551b292f8c3c3d0bb0539825ba/tensorflow/python/util/lazy_loader.py#L129

Reply:
Done